addons/movments_accounts_balances/controllers/controllers.py

```diff
@@ -79,3 +97,183 @@ def general_ledger_report(self, account_id, start_date, end_date):
             'ledger_data': ledger_data,
         }
 
+    @api.model
+    def get_bills(self, start_date, end_date, selected_account_id, selected_partner_id, selected_analytic_id):
```

Comment:
@sameerz09 we will need to pass in the journal_entry id. For all the objects, we will do the same - for "get and delete" methods, we will pass in journal_entry id. Try this method:
```@api.model
    def get_bill(self, journal_entry_id):
        Bill = self.env['account.move']
        bill = Bill.search([('id', '=', journal_entry_id), ('move_type', '=', 'in_invoice')])
        return bill